CONTRIBUTING.md

```diff
@@ -76,12 +76,13 @@ Some test take more time to run (run a simulation) and were therefore disabled b
  nevertheless important that you run a simulation test before you ask a review, or whenever you
   are ready to merge, to make sure the
   code still works.
+  
  ```bash
-EXECUTE_TESTS_ON=dev pytest
+EXECUTE_TESTS_ON=dev pytest tests/test_benchmark_simple_scenarios.py
```

Comment:
Why do you want to only run this test? and not all tests?

Reply:
I don´t know, tests usually do not run through on my laptop anyway (networkx plot issue, mainly), so I could not test it. We can change it to the whole `tests` folder?

Reply:
If you run `pytest` it will run all modules starting with `test_` in all subdirectories from the root of the repo. As we only have tests modules in `tests` folder, `pytest` or `pytest tests` is equivalent. On the otherside `pytest tests/test_benchmark_simple_scenarios.py` will *only* run `test_benchmark_simple_scenarios.py` and none of the other tests